modules/cluster-logging-cloudwatch.adoc

```diff
@@ -6,10 +6,6 @@
 // can be re-used in associated products.
 
 :_mod-docs-content-type: CONCEPT
-[id="cluster-logging-cloudwatch_{context}"]
-= CloudWatch recommendation for {product-title}
-
-Red Hat recommends that you use the AWS CloudWatch solution for your logging needs.
```

Comment:
Yes, let's completely remove this sentence as this is too specific to AWS. 

Instead, can we point users to [Log collection and forwarding](https://docs.openshift.com/dedicated/observability/logging/log_collection_forwarding/log-forwarding.html) section to learn more about "Log Output Types", "Configuring Log Forwarding" and other topics related to this?

[1] [https://docs.openshift.com/dedicated/observability/logging/log_collection_forwarding/logging-output-types.html](https://www.google.com/url?q=https://docs.openshift.com/dedicated/observability/logging/log_collection_forwarding/logging-output-types.html&sa=D&source=docs&ust=1715298068385815&usg=AOvVaw3xpxMVNbm6XxPltCcNmmS9)

[2] [https://docs.openshift.com/dedicated/observability/logging/log_collection_forwarding/configuring-log-forwarding.html](https://www.google.com/url?q=https://docs.openshift.com/dedicated/observability/logging/log_collection_forwarding/configuring-log-forwarding.html&sa=D&source=docs&ust=1715298068385923&usg=AOvVaw1wcBLXVuU6BPBDKX139rsN)

Reply:
I'll add a "For information, see..." link to Log collection and forwarding.